etl/steps/data/garden/wb/2023-07-10/education.py

```diff
@@ -0,0 +1,153 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from typing import cast
+
+from owid.catalog import Dataset, Table
+from owid.catalog.utils import underscore
+from tqdm import tqdm
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+
+    # Load meadow dataset.
+    ds_meadow = cast(Dataset, paths.load_dependency("education"))
+
+    # Read table from meadow dataset.
+    tb = ds_meadow["education"]
+    tb.reset_index(inplace=True)
```

Comment:
This line and the ones below are already data processing steps, so they should be under "Process data", for consistency.